WordPressAuthenticator/Extensions/UIStoryboard+Helpers.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+
+// MARK - Storyboard enum
+enum Storyboard: String {
+    case login
+    case signup
+
+    var instance: UIStoryboard {
+        return UIStoryboard(name: self.rawValue.capitalized, bundle: WordPressAuthenticator.bundle)
```

Comment:
`WordPressAuthenticator.swift` determines which bundle to use based on `If installed via CocoaPods`. This does not. Does it need to?

Reply:
I'm confused. `WordPressAuthenticator.bundle` calls the [member class variable bundle](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/develop/WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift#L439). The comment says it determines which bundle to use. Is calling the getter wrong? I don't understand bundles very well. Is this line supposed to be plain `bundle`?

Reply:
🤦‍♂ sorry. I misread what that was doing. Nothing to see here...

Reply:
:blob-whew: